src/test/java/com/rakovets/course/java/core/StringUtilTest.java

```diff
@@ -0,0 +1,320 @@
+package com.rakovets.course.java.core;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+class StringUtilTest {
+
+    static Stream<Arguments> Test1() {
```

Comment:
The name is WRONG for method.
```suggestion
    static Stream<Arguments> getConcatProviderArguments() {
```